tdvt/tdvt/config_gen/datasource_list.py

```diff
@@ -327,7 +327,7 @@ def load_test(config, test_dir=get_root_dir()):
 
             try:
                 all_ini_sections.remove(section)
-                test_config.add_expression_test('StaplesConnectionTest', STAPLES_TDS, sect.get(KEY_EXCLUSIONS, ''),
+                test_config.add_logical_test('StaplesConnectionTest', STAPLES_TDS, sect.get(KEY_EXCLUSIONS, ''),
```

Comment:
So this is needed to account for the different LogicalQueryFormat configurations?

Reply:
The answer is apparently yes, please note in the item which config is used and the partner details.